src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -662,10 +716,47 @@ public boolean isInGoal(int x, int y, int z) {
         }
         if (msg.equals("damn")) {
             logDirect("daniel");
+            return true;
+        }
+        if (msg.startsWith("add")){
+            String newCommands = msg.substring(3).trim();
+            commandsToBeExecuted = commandsToBeExecutedOld+ newCommands;
+            return true;
+        }
+        if (msg.equals("next")){
+            commandsToBeExecuted = commandsToBeExecutedOld;
+            nextCommand();
+
+            return true;
+        }
+        if (msg.equals("commandlist")){
+            commandsToBeExecuted = commandsToBeExecutedOld;
+            logDirect(commandsToBeExecuted);
+            return true;
         }
         return false;
     }
 
+    private static ExampleBaritoneControl thisExampleBaritoneControl;
```

Comment:
fields go at the top. this field needs to be killed and the other two need to be non static